doc/source/faq.rst

```diff
@@ -43,16 +43,28 @@ Similarly, if you have many workers processing large chunks of data you may
 be using much more memory than you expect. If you limit the number of workers
 *and* the size of the data chunks being processed by each worker you can
 reduce the overall memory usage. Default chunk size can be configured in Satpy
-by setting the following environment variable:
+by using the following around your code:
 
-.. code-block:: bash
+.. code-block:: python
+
+    with dask.config.set("array.chunk-size": "32MiB"):
+      # your code here
+
+The value of the chunk-size can be given in different ways, see here:
+https://docs.dask.org/en/stable/api.html#dask.utils.parse_bytes
+There are other ways to set dask configuration items, including configuration
+files or environment variables, see here:
+https://docs.dask.org/en/stable/configuration.html
+
+The default value for this parameter is 128MiB, which can translate to chunk
+sizes of 4096x4096 for 8-byte float arrays. Note however that readers might
+interpret this value to create non-square chunks for better performance.
+
+.. note::
 
-    export PYTROLL_CHUNK_SIZE=2048
+    The PYTROLL_CHUNK_SIZE variable is pending deprecation, so the
+    above-mentioned dask configuration parameter should be used instead.
```

Comment:
Can we please document a way to do this outside code?  I use this environment variable in my supervisor configuration, which calls trollflow2, which calls satpy.  This never reached custom code.